src/test/util/json.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <test/util/json.h>
+
+#include <boost/test/unit_test.hpp>
+#include <string>
+
+#include <univalue.h>
+
+UniValue read_json(const std::string& jsondata)
+{
+    UniValue v;
+    if (!v.read(jsondata) || !v.isArray()) {
+        BOOST_ERROR("Parse error.");
```

Comment:
In theory it is not allowed to link boosttest to libtest_util, but given that CI didn't fail, `BOOST_ERROR` is probably header-only?

Reply:
39e66e938fb688f5400ad94a1b317fcc2a87bc31 from #24301 ?

Reply:
Ah ok. I think I have a slight preference to keep boost out of libtest_util, and thus out of the fuzz tests

Reply:
What about just moving the `read_json` function into the `src/test/main.cpp` source file?

Reply:
> Ah ok. I think I have a slight preference to keep boost out of libtest_util, and thus out of the fuzz tests

We already have: https://github.com/bitcoin/bitcoin/blob/ea67232cdb80c4bc3f16fcd823f6f811fd8903e1/src/test/util/chainstate.h#L17

Reply:
We can probably remove that, and keep it Boost free.

Reply:
> We already have: 

This can probably be replaced by a LogPrint or so

Reply:
Done in #26009.

Reply:
Now that #26009 has been merged, this should be reworked to avoid Boost usage.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/25974#issuecomment-1253909158).